konf/site.yaml

```diff
@@ -60,6 +60,16 @@ env:
     secretKeyRef:
       key: password
       name: badgr
+  
+  - name: BAGDR_QA_USER
```

Comment:
You define this, but you never you it because you hard-code it in the decorator.

Reply:
Yeah, I was thinking maybe we need to add a different user? But maybe that's not up to us. So removed.